### PR TITLE
Add navigation

### PR DESCRIPTION
**Pull request by svenvonwerner**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -37,4 +49,15 @@ function App() {
   }
 }
 
+const AppGrid = styled.div`
+  display: grid;
+  grid-template-rows: 48px 1 fr 48px;
```
Comment (macfrei) [inline src/App.js:54]:
```suggestion
  grid-template-rows: 48px 1fr 48px;
```

File: src/components/Form.js [hunk 2]
```diff
@@ -3,7 +3,11 @@ import styled from 'styled-components';
...
 
 export default function Form({ handleCreateCard }) {
-  const { register, handleSubmit } = useForm();
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+  } = useForm();
 
   // const onSubmit = data => console.log(data);
```
Comment (macfrei) [inline src/components/Form.js:12]: You can delete this comment. There is another comment in this component. I can't comment there but it should be deleted.

**Review by MaggieSchuetz — APPROVED** [review]
Looks great! No complaints :)

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  background-color: white;
+  margin: 0px;
+  justify-content: center;
+  width: 100%;
+  margin-top: 0px;
+  position: fixed;
+  top: 0;
+`;
+
+const Headline = styled.p`
```
Comment (macfrei) [inline src/components/Header.js:25]: Usually a headline is something like `h1 ~ h6`

File: src/components/Navigation.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  color: white;
+  justify-content: space-around;
+  align-items: center;
+  box-shadow: 0px -1px 2px 0px rgba(0, 0, 0, 0.5);
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+`;
+
+const NavButton = styled(NavLink)``;
```
Comment (macfrei) [inline src/components/Navigation.js:33]: If this is empty, then you don't need do use the styled-components syntax.

File: src/components/Navigation.spec.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+import Navigation from './Navigation.js';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('Form', () => {
+  it('renders two navigation buttons', () => {
+    render(
+      <MemoryRouter>
+        {' '}
```
Comment (macfrei) [inline src/components/Navigation.spec.js:9]:
```suggestion
```

File: src/components/Navigation.spec.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+import Navigation from './Navigation.js';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('Form', () => {
+  it('renders two navigation buttons', () => {
```
Comment (macfrei) [inline src/components/Navigation.spec.js:6]: It actually renders two links and not buttons.
```suggestion
  it('renders two navigation links', () => {
```

File: src/components/Navigation.spec.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+        {' '}
+        <Navigation />
+      </MemoryRouter>
+    );
+
+    const button1 = screen.getByAltText('Switch to Form');
+    const button2 = screen.getByAltText('Switch to List');
+
+    expect(button1).toBeInTheDocument();
+    expect(button2).toBeInTheDocument();
```
Comment (macfrei) [inline src/components/Navigation.spec.js:18]:
```suggestion
    const formIcon = screen.getByAltText('Switch to Form');
    const listIcon = screen.getByAltText('Switch to List');

    expect(formIcon).toBeInTheDocument();
    expect(listIcon).toBeInTheDocument();
```

File: src/pages/ChallengeListPage.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+export default function ChallengeListPage({ handleCheckClick, challengeData }) {
+  return (
+    <>
+      <Subheadline>My Challenge Photo List</Subheadline>
+      <ListCard handleCheckClick={handleCheckClick} challengeData={challengeData} />
+    </>
+  );
+}
+
+const Subheadline = styled.h1`
```
Comment (macfrei) [inline src/pages/ChallengeListPage.js:13]: Usually `h1` is not a subheadline. Maybe change the name here?

File: src/pages/ChallengeListPage.js [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+import styled from 'styled-components';
+
+export default function ChallengeListPage({ handleCheckClick, challengeData }) {
+  return (
+    <>
+      <Subheadline>My Challenge Photo List</Subheadline>
+      <ListCard handleCheckClick={handleCheckClick} challengeData={challengeData} />
+    </>
+  );
+}
```
Comment (macfrei) [inline src/pages/ChallengeListPage.js:11]:
```suggestion
export default function ChallengeListPage({ onCheckClick, challengeData }) {
  return (
    <>
      <Subheadline>My Challenge Photo List</Subheadline>
      <ListCard onCheckClick={onCheckClick} challengeData={challengeData} />
    </>
  );
}
```

File: src/pages/CreateChallengePage.js [hunk 10]
```diff
@@ -0,0 +1,17 @@
+import Form from '../components/Form.js';
+import styled from 'styled-components';
+
+export default function CreateChallengePage({ handleCreateCard }) {
+  return (
+    <div>
+      <Subheadline>Create a Photo Challenge</Subheadline>
+      <Form handleCreateCard={handleCreateCard} />
+      {/* <Navigation /> */}
```
Comment (macfrei) [inline src/pages/CreateChallengePage.js:9]: Please remove this comment :)

**Review by Quaquaro — APPROVED** [review]

**Pull request merged**